stable/datadog/README.md

```diff
@@ -106,6 +106,23 @@ datadog:
  logsConfigContainerCollectAll: true
 ```
 
+To collect custom logs, the following configuration could be used:
```

Comment:
could you document the fact that if the file is not in the agent container, they must mount the host volume to it thanks to:
```
datadog:
  ## @param volumes - list of objects - optional
  ## Specify additional volumes to mount in the dd-agent container
  #
  # volumes:
  #   - hostPath:
  #       path: <HOST_PATH>
  #     name: <VOLUME_NAME>

  ## @param volumeMounts - list of objects - optional
  ## Specify additional volumes to mount in the dd-agent container
  #
  # volumeMounts:
  #   - name: <VOLUME_NAME>
  #     mountPath: <CONTAINER_PATH>
  #     readOnly: true
```